deployments/kovan/usdc/deploy.ts

```diff
@@ -1,251 +1,176 @@
-import { DeploymentManager } from '../../../plugins/deployment_manager/DeploymentManager';
-import { deployComet } from '../../../src/deploy';
-import { exp, wait } from '../../../test/helpers';
-import {
-  Bulker,
-  Bulker__factory,
-  Fauceteer,
-  Fauceteer__factory,
-  ProxyAdmin,
-  ProxyAdmin__factory,
-} from '../../../build/types';
-import { Contract } from 'ethers';
-import { debug } from '../../../plugins/deployment_manager/Utils';
-
-let cloneNetwork = 'mainnet';
-let cloneAddr = {
-  usdcImplementation: '0xa2327a938Febf5FEC13baCFb16Ae10EcBc4cbDCF',
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { Bulker, Fauceteer, ProxyAdmin } from '../../../build/types';
+import { DeploySpec, debug, deployComet, exp, getBlock, sameAddress, wait } from '../../../src/deploy';
+
+const cloneNetwork = 'mainnet';
+const cloneAddr = {
+  usdcImpl: '0xa2327a938Febf5FEC13baCFb16Ae10EcBc4cbDCF',
   usdcProxy: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
   wbtc: '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599',
   weth: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
   comp: '0xc00e94cb662c3520282e6f5717214004a7f26888',
-  uni: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
   link: '0x514910771af9ca656af840dff83e8264ecf986ca',
+  uni: '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984',
 };
 
-interface Vars {
-  comet: string,
-  configurator: string,
-  fauceteer: string,
-  rewards: string,
-  bulker: string
-};
-
-export default async function deploy(deploymentManager: DeploymentManager) {
-  return await deploymentManager.doThenVerify(
-    () => deployAll(deploymentManager)
-  );
-}
-
-async function deployAll(deploymentManager: DeploymentManager) {
-  const newRoots = await deployContracts(deploymentManager);
-  deploymentManager.putRoots(new Map(Object.entries(newRoots)));
```

Comment:
Curious where roots are set now? Is it implicitly being set?

Reply:
Only after you spider